airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -538,7 +542,7 @@ def sasl_factory():
 
         return hmsclient.HMSClient(iprot=protocol)
 
-    def _find_valid_server(self):
+    def _find_valid_server(self):  # pylint: disable=inconsistent-return-statements
```

Comment:
I would prefer to fix it instead of disabling it. `return None` at the end should be enough :)

Reply:
Fix